journal.bib

```diff
@@ -49,17 +50,19 @@ @string{ IEEETSMCB
 @string{ IPV         = "IFAC Proceedings Volumes" }
 @string{ IJARS       = "International Journal of Advanced Robotic Systems" }
 @string{ IJICC       = "International Journal of Intelligent Computing and Cybernetics" }
+@string{ IJS         = "Sankhyā: The Indian Journal of Statistics" }
 @string{ ITIT        = "it - Information Technology" }
 @string{ JA          = "Journal of Algorithms" }
 @string{ JAIR        = "Journal of Artificial Intelligence Research"}
+@string{ JASA        = "Journal of the American Statistical Association" }
```

Comment:
We are organizing the journals with respect to their proper name and not with respect to their key. This should go after Journal of Systems Architecture.